docs/src/modules/components/MarkdownDocs.js

```diff
@@ -80,6 +80,9 @@ export default function MarkdownDocs(props) {
               throw new Error(`No component found at the path ${path.join('docs/src', name)}`);
             }
 
+            if (renderedMarkdownOrDemo.skipBrandingProvider) {
+              return <Component {...renderedMarkdownOrDemo} markdown={localizedDoc} />;
+            }
```

Comment:
I have to add this to make the Templates free from BrandingProvider (Material UI's Provider).